static/_redirects

```diff
@@ -181,9 +181,11 @@
 /docs/changelog.html /docs/overview/changelog
 
 /docs/reference/identity-provider-service-account /docs/releases/upgrading
-/docs/reference/cookie-secure /docs/reference/cookie-secret-file
 /reference/ /docs/reference/
 /reference/* /docs/reference/:splat
+/docs/reference/cookie-secure /docs/reference/https-only
+/docs/reference/cookie-http-only /docs/reference/expiration
+/docs/reference/cookie-expire /docs/reference/expiration
```

Comment:
This... should probably be named cookie-expire like it was originally. not sure why it was renamed. 

https://www.pomerium.com/docs/reference/expiration